test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_core_test.clj

```diff
@@ -189,125 +192,139 @@
 
 (defn get-webserver-context-for-server
   []
-  {:state (atom nil)
-   :handlers
-           (ContextHandlerCollection.)
-   :server nil})
-
-(defn get-http-config-for-server
-  [max-threads queue-max-size so-linger-milliseconds idle-timeout-milliseconds]
-  {:http {:port 0
-          :host "0.0.0.0"
-          :request-header-max-size 100
-          :so-linger-milliseconds so-linger-milliseconds
-          :idle-timeout-milliseconds idle-timeout-milliseconds}
-   :jmx-enable false
-   :max-threads max-threads
-   :queue-max-size queue-max-size})
-
-(defn get-thread-pool-and-queue-for-create-server
-  [max-threads queue-max-size]
-  (let [server           (jetty/create-server
-                           (get-webserver-context-for-server)
-                           (get-http-config-for-server max-threads
-                                                       queue-max-size
-                                                       0
-                                                       nil))
-        thread-pool      (.getThreadPool server)
-        ;; Using reflection here because the .getQueue method is protected
-        ;; and I didn't see any other way to pull the queue back from
-        ;; the thread pool.
-        get-queue-method (-> thread-pool
-                             (.getClass)
-                             (.getDeclaredMethod "getQueue" nil))
-        _                (.setAccessible get-queue-method true)
-        queue            (.invoke get-queue-method thread-pool nil)]
-    {:thread-pool thread-pool
-     :queue queue}))
-
-(deftest create-server-thread-pool-test
```

Comment:
These tests still effectively exist under `create-server-queue-max-size-test` but they've been refactored to eliminate some duplication and also validate Jetty defaults when not explicitly set in user configuration.